backend/src/Edna/Analysis/FourPL.hs

```diff
@@ -66,7 +65,7 @@ instance ToSchema Params4PL where
 type AnalysisResult = Either Text Params4PL
 
 instance Buildable (AnalysisResult) where
-  build = build . (coerce :: AnalysisResult -> Either Text Params4PL)
+  build = build
```

Comment:
And now it's infinite recursion.

Reply:
But it works 🤔

Reply:
No, did you try calling `/new/analyse` endpoint? It hangs.

Reply:
Okay, I tested just `new`, not `/new/analyse`. `build . id` also doesn't work. I maybe don't understand smth, but why I need instance for type synonym?

Reply:
You need instance for `Either Text Params4PL`, it simply doesn't exist.

Reply:
:fffuuu: \
Fixed